torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -18,16 +18,29 @@ class LibKinetoClient : public libkineto::ClientInterface {
     reportInputShapes_ = setupOpInputsCollection;
   }
 
+#ifdef USE_KINETO_UPDATED
+  void start(bool withStack) override {
```

Comment:
You can reduce the size of the `#ifdef` block:
```
#ifdef VERSION_GUARD_VAR
  void start(bool withStack) override {
#else
  void start() override {
    bool withStack{false};
#endif
```

Reply:
as mentioned above. this guard will be removed at the end.